Add warning regarding "class not found" messages when using packages (#601)
projectsyn/commodore: docs/modules/ROOT/pages/tutorial/package.adoc

This will add the component cert-manager to the cluster and configure the lestsencrypt mail to `cert@example.com`.

[NOTE]
====

> During catalog compilation, you might see several warnings like `[WARNING] Reclass class not found: 'cert-manager.common'. Skipped!`.

Reviewer: ```suggestion
During catalog compilation, after the message `Discovering config packages...`, you'll see several warnings like `[WARNING] Reclass class not found: 'cert-manager.common'. Skipped!`.
```

> This is expected, as Commodore first has to render the inventory to even be able to clone packages.

Reviewer: ```suggestion
This is expected, as Commodore has to render the inventory in order to be able to read the package's Git repository URL and version, which are stored in the inventory.
```

> Still, double-check the manifests to ensure the parameters really get applied.

Reviewer: ```suggestion
```

If you include a class which isn't found after making packages available, the compilation will fail, so if compilation succeeds with the package included, parameters defined in the package class are always applied.
